__tests__/app.test.js

```diff
@@ -18,16 +18,19 @@ describe("GET topics", () => {
       .get("/api/topics")
       .expect(200)
       .then((res) => {
+        let count = 0;
         expect(Array.isArray(res.body.topics)).toBe(true);
         res.body.topics.forEach((topics) => {
+          count++;
           expect(typeof topics.slug).toBe("string");
           expect(typeof topics.description).toBe("string");
         });
+        expect(count).toBe(3);
```

Comment:
checking the length of the topics array would be better than a count

Reply:
just seen that this is for a previous ticket though, make sure you stick to code for this ticket on each PR